docs/HowTo/Get-Started/Run-Teku.md

```diff
@@ -15,6 +15,11 @@ processes.
 We recommend you run the beacon chain client and validator as a [single process] if they are to run
 on the same machine.
 
+!!! note
+
+    By default, Teku connects to `mainnet`. Use the [`--network`](../../Reference/CLI/CLI-Syntax.md#network)
+    command line option to specify an alternate network.
```

Comment:
We might need to note that because the mainnet genesis state is not yet known, Teku requires an `--eth1-endpoint` to be specified when running with MainNet.

Reply:
Done